examples/lock-app/esp32/CMakeLists.txt

```diff
@@ -27,5 +27,18 @@ set(EXTRA_COMPONENT_DIRS
 )
 
 project(chip-lock-app)
-idf_build_set_property(CXX_COMPILE_OPTIONS "-std=c++14;-Os;-DLWIP_IPV6_SCOPES=0;-DCHIP_HAVE_CONFIG_H" APPEND)
+idf_build_set_property(CXX_COMPILE_OPTIONS "-std=c++17;-Os;-DLWIP_IPV6_SCOPES=0;-DCHIP_HAVE_CONFIG_H" APPEND)
 idf_build_set_property(C_COMPILE_OPTIONS "-Os;-DLWIP_IPV6_SCOPES=0" APPEND)
+
+if (CONFIG_ENABLE_PW_RPC)
+# C++17 is required for RPC build.
```

Comment:
Should this comment be around the `-std=c++17` present above?

Reply:
Done